iothub_client/src/iothub_client_ll_uploadtoblob.c

```diff
@@ -908,6 +914,8 @@ void IoTHubClient_LL_UploadToBlob_Destroy(IOTHUB_CLIENT_LL_UPLOADTOBLOB_HANDLE h
         {
             free(upload_data->credentials.x509_credentials.x509certificate);
             free(upload_data->credentials.x509_credentials.x509privatekey);
+            free(upload_data->credentials.x509_credentials.x509privatekeyType);
```

Comment:
These two `free` calls will require additions in `IoTHubClient_LL_UploadToBlob_Destroy_handle_x509_succeeds` which currently doesn't expect them.